services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DigitalCertificatesStructureProvider.java

```diff
@@ -144,7 +148,9 @@ private DigitalCertificatesDirectory constructArchiveToPublish(DigitalGreenCerti
 
     try {
       rulesArchive
-          .addWritable(new FileOnDisk("export.bin", dgcToCborMapping.constructCborRules(ruleType)));
+          .addWritable(new FileOnDisk("export.bin", dgcToCborMapping
```

Comment:
If dgcToCborMapping allows the use of suppliers now, why not to pull out this method in another class, add the suppliers in parameters and just reference the same method both in DigitalCertificatesStructureProvider and BoosterNotificationStructureProvider ?

Reply:
Please create a refactoring ticket for this one.